appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
   - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - ruby --version
   - gem --version
-  - gem install bundler
+  - gem install bundler -v 1.11.2
```

Comment:
I guess it's not clear if it was a fluke or if bundler 1.12 is broken on Windows.  Since we're not sure, let's not push this to the template just yet.  Can you add a comment explaining that, though?

If we see other builds fail with bundler 1.12 we can add it to the template; or if we see other builds pass, this'll get blown away the next time we re-gen from the template.